src/lib/cache/cache.hpp

```diff
@@ -78,6 +78,8 @@ class Cache {
 
   Iterator end() { return _impl->end(); }
 
+  size_t frequency(const Key& key) { return _impl->frequency(key); }
```

Comment:
doc?

Reply:
What about:
"Returns the frequency with which a cached item has been accessed (=1 for set(), +1 for each get()). Returns 0 for keys not cached." (second part is Jan's remark, working on it right now).

Reply:
`access frequency` instead of `with which...`?

Reply:
"Returns the access frequency of a cached item (=1 for set(), +1 for each get()). Returns 0 for keys not being cache-resident."

?

Reply:
works for me

Reply:
Added the comment.